x-pack/plugins/reporting/server/browsers/index.ts

```diff
@@ -42,7 +41,7 @@ export const initializeBrowserDriverFactory = async (
   config: ReportingConfig,
   logger: LevelLogger
 ) => {
-  const { binaryPath$ } = installBrowser(chromium, config, logger);
+  const { binaryPath$ } = installBrowser(logger);
```

Comment:
We should probably rename `installBrowser` since it's not doing any installs anymore